README.md

```diff
@@ -19,20 +19,29 @@ Currently supported platforms are Windows and Linux, x86_64 only.
 ### Required for Windows:
 * `python` installed
 * `cmake`
-* Visual Studio compilers, version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+* Visual Studio with release year not older than 2015 and Visual Studio compiler, which version depends on which Python version you want to target, see [correct MSVC version](https://wiki.python.org/moin/WindowsCompilers)
+
 
 
 ### Before building do:
 * On Linux:
   * `git submodule update --init --recursive`
   * `cd 3rd_party && ./prereq-build.sh && cd ..`
 * On Windows:
-  * `git submodule update --init --recursive`
-  * `cd 3rd_party/protobuf-c && git apply ../protobuf-c-vs2008-support.patch && cd ../..`
+  * Update third-party intel-xed, mbuild, protobuf, protobuf-c and safestringlib submodules (3rd_party folder):
+      * `git submodule update --init -- .\3rd_party\intel-xed`
+      * `git submodule update --init -- .\3rd_party\mbuild`
+      * `git submodule update --init -- .\3rd_party\protobuf`
+      * `git submodule update --init -- .\3rd_party\protobuf-c`
+      * `git submodule update --init -- .\3rd_party\safestringlib`
+  * Apply protobuf-c-vs2008-support.patch:
```

Comment:
add a note that this is required for Python 2 target only (not needed for Python 3)

Reply:
Ok, comment was addressed in new commit.